package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
i think the ideal solution to this is (and my personal preference), is to use `npm link`.

i have `nodeapi/` and `halfpenny/` on my machine.  i ran `npm link` in halfpenny's dir, then `npm link halfpenny` in nodeapi's dir.  now, nodeapi always looks my my local mahcine's halfpenny folder instead of it's own node_modules.  really, i think nodeapi/node_modules/halfpenny is a symlink back to halfpenny, but it's managed.

this is how I did my dev in the feature, and it was _very, very pleasant_!

what are your thoughts?